src/index.html

```diff
@@ -16,7 +37,496 @@
+            class="explore-online"
+            id="explore-online"
+          >
+            <h2 class="section-title">
+              More to Explore
+              <br />
+              online
+            </h2>
+
+            <div class="explore-online__tours">
+              <article class="explore-online__tour">
+                <a href="#">
+                  <img
+                    src="images/page/explore-online-01.png"
+                    alt="new-look"
+                    class="explore-online__photo"
+                  />
+                </a>
+                <h3 class="explore-online__title">Collection Areas</h3>
+                <p class="explore-online__description">
+                  These collection areas study, exhibit, and care for the
+                  objects in The Met collection.
+                </p>
+              </article>
+
+              <article class="explore-online__tour">
+                <a href="#">
+                  <img
+                    src="images/page/explore-online-02.png"
+                    alt="new-look"
+                    class="explore-online__photo"
+                  />
+                </a>
+                <h3 class="explore-online__title">Travel</h3>
+                <p class="explore-online__description">
+                  Travel with The Met connects the Museum's wide and diverse
+                  audience to art, inspiration, and historical perspective.
+                </p>
+              </article>
+
+              <article class="explore-online__tour">
+                <a href="#">
+                  <img
+                    src="images/page/explore-online-03.png"
+                    alt="new-look"
+                    class="explore-online__photo"
+                  />
+                </a>
+                <h3 class="explore-online__title">Virtual group tours</h3>
+                <p class="explore-online__description">
+                  Request a guided tour, reserve admission and lead your own
+                  small group, or request a live virtual tour led by a Museum
+                  guide.
+                </p>
+              </article>
+            </div>
+          </section>
+          <!-- Contacts -->
+          <section
+            class="contacts"
+            id="contacts"
+          >
+            <h2 class="section-title">Contact us</h2>
+            <div class="contacts__content">
+              <div class="contacts__info">
+                <div>
+                  <h3>The Met Fifth Avenue</h3>
+                  <h4 class="contacts__label">Phone</h4>
+                  <a
+                    class="contacts__link"
+                    href="tel:+12125357710"
+                  >
+                    +1 212-535-7710
+                  </a>
+                  <h4 class="contacts__label">Address</h4>
+                  <a
+                    class="contacts__link"
+                    href="https://maps.app.goo.gl/CCHuCF4PC2njcLEY7"
```

Comment:
These links should lead these location's google maps page not a random place

Reply:
Done.